sdk/batch/Microsoft.Azure.Batch.FileStaging/src/Microsoft.Azure.Batch.FileStaging.csproj

```diff
@@ -17,19 +17,25 @@
     </PackageReleaseNotes>
   </PropertyGroup>
   <PropertyGroup>
-    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
   </PropertyGroup>
-
   <ItemGroup>
-    <PackageReference Include="System.Text.Encodings.Web" />
-    <PackageReference Include="WindowsAzure.Storage" />
+    <PackageReference Include="System.Text.Encodings.Web" VersionOverride="4.7.2" />
     <PackageReference Include="Microsoft.Azure.Batch" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'net452' ">
     <Reference Include="System.Web" />
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
```

Comment:
These are the same references why split them?

Reply:
Good catch, pushed a change to consolidate the references.